### ISSUE TITLE
Improve lore2sphinx buildbot results for `projects/core/howto/testing.xhtml`

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#4874]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)| @exarkun reported|
|-|-|
|Trac ID|trac#4874|
|Type|enhancement|
|Created|2011-02-11 05:39:56Z|
|Branch|https://github.com/twisted/twisted/tree/testing-sphinx-4874|

like [#4566](https://github.com/twisted/twisted/issues/4566), except for the testing howtos

<details><summary>Searchable metadata</summary>

```
trac-id__4874 4874
type__enhancement enhancement
reporter__exarkun exarkun
priority__normal normal
milestone__ 
branch__branches_testing_sphinx_4874 branches/testing-sphinx-4874
branch_author__screwtape__jdb screwtape, jdb
status__closed closed
resolution__fixed fixed
component__core core
keywords__documentation documentation
time__1297402796000000 1297402796000000
changetime__1300248452000000 1300248452000000
version__None None
owner__exarkun exarkun

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_1"></a>|@exarkun commented|
|-|-|

(In [30680]) Branching to 'testing-sphinx-4874'

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_2"></a>|@exarkun commented|
|-|-|

(In [30681]) Merge the telnet parts of the impossibly huge [[#4568](https://github.com/twisted/twisted/issues/4568)](https://github.com/twisted/twisted/issues/4568) branch

refs [#4874](https://github.com/twisted/twisted/issues/4874)
refs #4568

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="washort's avatar" src="https://avatars.githubusercontent.com/u/947487?s=50" width="50" height="50">](https://github.com/washort)<a name="note_5"></a>|@washort set owner to @exarkun|
|-|-|

Looks good - renders correctly in sphinx and lore.

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_6"></a>|@exarkun set status to `closed`|
|-|-|

(In [31172]) Merge testing-sphinx-4874 

Author: screwtape, jdb
Reviewer: washort
Fixes: [#4874](https://github.com/twisted/twisted/issues/4874)

Fix various lore2sphinx conversion nits.
